src/plugins/plugins.h

```diff
@@ -238,6 +238,12 @@ class plugin
 {
 public:
     virtual ~plugin() = default;
+
+    virtual void stop() {}
+    virtual bool is_stopped()
+    {
+        return true;
```

Comment:
This doesn't seem right either, why is the default to return `true`?

Reply:
@tklengyel I'm pretty sure about default value `true`. Because this doesn't break current drakvuf logic.

Reply:
Thank You! I have fixed logical error.